GhostLegs/glw169/bones_sol.py

```diff
@@ -80,20 +72,16 @@
                     if bone[1] == idx and bone[0] not in moved:
```

Comment:
Odd indentation here.

Reply:
I do not understand your review; Python relies on indentation for control flow (`if`, `for`, ect...). I don't believe I can change the indentation without obfuscating the code into a one-liner.

Reply:
Perhaps you're using tabs instead of spaces? This is an indent of 8 spaces, not 4, while the rest of your code is indented with 4 spaces.